backend/audit/templates/audit/my_submissions.html

```diff
@@ -86,38 +78,38 @@ <h2 class="usa-modal__heading" id="modal-uei-info-heading">What is a Unique Enti
         </div>
     </div>
 
-    <div
-        class="usa-modal usa-modal--lg flex-justify-start"
-        id="modal-terms-conditions"
-        aria-labelledby="modal-terms-conditions-heading"
-        aria-describedby="modal-terms-conditions-description"
-    >
+    <div class="usa-modal usa-modal--lg flex-justify-start" id="modal-terms-conditions"
+        aria-labelledby="modal-terms-conditions-heading" aria-describedby="modal-terms-conditions-description">
         <div class="usa-modal__content">
             <div class="usa-modal__main">
-                <h2 class="usa-modal__heading" id="modal-terms-conditions-heading">Federal Audit Clearinghouse terms and conditions</h2>
+                <h2 class="usa-modal__heading" id="modal-terms-conditions-heading">Federal Audit Clearinghouse terms and
```

Comment:
This is another example of where splitting it into multiple lines doesn't seem correct.